dto/src/main/java/ca/on/oicr/gsi/runscanner/dto/dragen/DragenPipelineRun.java

```diff
@@ -0,0 +1,38 @@
+package ca.on.oicr.gsi.runscanner.dto.dragen;
+
+import ca.on.oicr.gsi.runscanner.dto.PipelineRun;
+import ca.on.oicr.gsi.runscanner.dto.dragen.samplesheet.Samplesheet;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import java.util.Objects;
+
+// Represents an entire attempt at a DRAGEN Pipeline
+public class DragenPipelineRun extends PipelineRun<DragenWorkflowRun> {
+  Samplesheet samplesheet;
```

Comment:
I understand the information stored in this `Samplesheet` object is needed during processing to check whether the workflows are complete, but should it be included in the output? It's mostly duplicating the information in `DragenAnalysisUnit`